erdpy/delegation/staking_provider.py

```diff
@@ -41,7 +43,9 @@ def prepare_args_for_add_nodes(args: Any):
         validator_pem = validator.get("pemFile")
         validator_pem = path.join(path.dirname(args.validators_file), validator_pem)
         seed, bls_key = parse_validator_pem(validator_pem)
-        signed_message = sign_message_with_bls_key(account.address.pubkey().hex(), seed.hex())
+        if not args.using_delegation_manager:
+            seed = seed.hex()
+        signed_message = sign_message_with_bls_key(account.address.pubkey().hex(), seed)
```

Comment:
Can you tell us how you tested this? The function `sign_message_with_bls_key()` expects ASCII arguments, because it will pass them to the MCL signer as a command line call. 

You probably wanted to switch between `account.address.pubkey().hex()` and `account.address.bech32()`, and leave the `seed` as it was. Was this your intention?

Reply:
I have tested in comparison with erdjs output (which I know is working correctly for the Delegation Manager).

The seed from `parse_validator_pem` is already in hex format. I don't know why hex() was called again on `seed`, but I don't want to break what was already depending on that implementation.

Reply:
But I see what you're saying. I'm not so experienced with python and I presume b'' is a binary representation of that String.
I guess somehow, [mcl_signer](https://github.com/ElrondNetwork/elrond-sdk-go-tools/blob/main/cmd/signer/mcl_signer.go) is reading the hex value correctly.

Reply:
In comparison, if you apply hex() again on the seed, you get a wrong seed sent to mcl_signer: `INFO:myprocess:run_process: ['/home/alex/elrondsdk/mcl_signer/v1.0.0/signer', '0000000000000000000100000000000000000000000000000000000006ffffff', '63623239316133336534346338373762653361653465336230383137666565383735383233646539306639656632323034356463633134343330663461343634']`

That's the seed hexed twice, I checked.

Reply:
Using the same seed for both cases now.